src/Controllers/CreateNewTaskDialogController.java

```diff
@@ -15,17 +15,10 @@
 	
 	@FXML Button btnCancel;
 	@FXML Button btnAccept;
```

Comment:
Can you add the type choice box here as well?

Reply:
Yes, please do.